java-compiler-testing/src/test/java/io/github/ascopes/jct/tests/unit/filemanagers/impl/JctFileManagerImplTest.java

```diff
@@ -76,4 +78,18 @@ void testAddPathForOutputLocation() {
     assertThat(jctFileManager.hasLocation(outputLocation)).isTrue();
   }
 
+  @Test
+  @DisplayName("Tests addPath method for adding module location")
+  void testAddPathForModuleLocation() {
+    var moduleLocation = StandardLocation.locationFor(StandardLocation.MODULE_PATH.name());
```

Comment:
Is this the same as just `var moduleLocation = StandardLocation.MODULE_PATH`?

Reply:
Thanks. I will make the change